Sources/ArcGISToolkit/Components/Augmented Reality/ARSwiftUIView.swift

```diff
@@ -153,3 +154,40 @@ class ARSwiftUIViewProxy: NSObject, ARSessionProviding {
         )
     }
 }
+
+extension ARSwiftUIViewProxy {
+    /// Performs a raycast to get the transformation matrix representing the corresponding 
+    /// real-world point for `screenPoint`.
+    /// - Parameter screenPoint: The screen point to determine the real world transformation matrix from.
+    /// - Returns: A `TransformationMatrix` representing the real-world point corresponding to `screenPoint`.
+    func raycast(from screenPoint: CGPoint) -> TransformationMatrix? {
```

Comment:
Since this method is used by both tabletop and world scale, I moved it here as an internal extension.